packages/x-charts/src/BarChart/BarPlot.tsx

```diff
@@ -63,33 +64,29 @@ export interface BarPlotProps extends Pick<BarElementProps, 'slots' | 'slotProps
     event: React.MouseEvent<SVGElement, MouseEvent>,
     barItemIdentifier: BarItemIdentifier,
   ) => void;
+  /**
+   * Defines the radius of the bar element.
```

Comment:
Nitpick: WDYT, would it make sense to include the `border` word in the description? 🤔 
```suggestion
   * Defines the border radius of the bar element.
```